website/pages/blog/azure-cost-optimization-with-cloudquery.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Correlating Costs to Resources for Azure Cost Optimization with CloudQuery
+tag: tutorial
+date: 2022/12/23
```

Comment:
This date should be edited to reflect the date of the blog, and should come after the Azure plugin release that contains https://github.com/cloudquery/cloudquery/pull/5898

Reply:
... which will be azure v3.1.0